GDJS/Runtime/object-capabilities/ScalableBehavior.ts

```diff
@@ -0,0 +1,112 @@
+/*
+ * GDevelop JS Platform
+ * Copyright 2013-2023 Florian Rival (Florian.Rival@gmail.com). All rights reserved.
+ * This project is released under the MIT License.
+ */
+namespace gdjs {
+  export interface Scalable {
+    /**
+     * Change the scale on X and Y axis of the object.
+     *
+     * @param newScale The new scale (must be greater than 0).
+     */
+    setScale(newScale: number): void;
+
+    /**
+     * Change the scale on X axis of the object (changing its width).
+     *
+     * @param newScale The new scale (must be greater than 0).
+     */
+    setScaleX(newScale: number): void;
+
+    /**
+     * Change the scale on Y axis of the object (changing its height).
+     *
+     * @param newScale The new scale (must be greater than 0).
+     */
+    setScaleY(newScale: number): void;
```

Comment:
I saw you added a Base3DHandler! Great! I'm not sure why it's not declared as the other handlers/capabilities though (same files and things). Because it's a JS only extension?

Reply:
Yes, objects that uses it are only in the 3D extension so I put it here.

I hesitated to split the 3D capability in smaller ones because maybe there will be 3D objects that don't have a size one day, but I guess it's not a big deal to have a scale action on a point light object for instance. I think it's better to keep it simple.